.nvmrc

```diff
@@ -1 +1 @@
-18
+20
```

Comment:
**suggestion:** Let's set this to the LTS version so we can always stay up-to-date.

```suggestion
lts/*
```
https://github.com/nvm-sh/nvm?tab=readme-ov-file#long-term-support